### ISSUE TITLE
Newick Data Read Incorrectly

### ISSUE THREAD

**Issue by mexiquin** [Zsailer/phylopandas#30]
I'm running data into a read_newick method and receive an error:
`unsupported operand type(s) for +: 'NoneType' and 'float'`

I only seem receive this error anytime I use one of my newick files (eg. ((A,B),(C,D));)

However, when I use the newick file provided inside of the examples folder, I don't receive the error.

Here's my traceback:
```
Traceback:

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Users/qjasper/Desktop/TestSite/foliumMap/testmap/views.py" in mapView
  21. 	newick = ppd.read_newick(data="((A,B),(C,D));").to_json()

File "/usr/local/lib/python3.7/site-packages/phylopandas/treeio/read.py" in func
  205.             **kwargs

File "/usr/local/lib/python3.7/site-packages/phylopandas/treeio/read.py" in _read
  156.         use_uids=use_uids

File "/usr/local/lib/python3.7/site-packages/phylopandas/treeio/read.py" in _dendropy_to_dataframe
  41.     tree.max_distance_from_root()

File "/usr/local/lib/python3.7/site-packages/dendropy/datamodel/treemodel.py" in max_distance_from_root
  5744.         dists = self.calc_node_root_distances()

File "/usr/local/lib/python3.7/site-packages/dendropy/datamodel/treemodel.py" in calc_node_root_distances
  5683.                 node.root_distance = node.edge.length + node._parent_node.root_distance

Exception Type: TypeError at /
Exception Value: unsupported operand type(s) for +: 'NoneType' and 'float'
```

**Comment by Zsailer** [comment]
@mexiquin thanks for posting this error. Looks like there is a bug in how we handle `null` values for branch lengths. Working on a fix now.

**Comment by Zsailer** [comment]
Fixed in #31. 

This issue was auto-closed by merging that pull-request. If this doesn't solve your issue, feel free to reopen the issue!
